Demo/src/main/java/com/paypal/android/ui/paypalwebvault/PayPalWebVaultView.kt

```diff
@@ -28,7 +32,7 @@ import com.paypal.android.utils.UIConstants
 import com.paypal.android.utils.getActivityOrNull
 
 @Composable
-fun PayPalWebVaultView(viewModel: PayPalWebVaultViewModel = hiltViewModel()) {
+fun PayPalWebVaultView(viewModel: PayPalVaultViewModel = hiltViewModel()) {
```

Comment:
```suggestion
fun PayPalVaultView(viewModel: PayPalVaultViewModel = hiltViewModel()) {
```
I like shortening the prefix. We'll have to look into doing this in the SDK as well. I think it makes sense to simplify the naming scheme as much as possible.